### ISSUE TITLE
Way to inject data into the Vuetables before first request on create?

### ISSUE THREAD

**Issue by lrembacz** [ratiw/vuetable-2#319]
Hi, Im doing session / filters recovery methods, that after redirect back Im getting all filters / pagers etc. in the same place I got it before i redirect to another page. Is there a way to inject this data back before request? And then get the same response as before?

**Comment by ratiw** [comment]
You may need to use js localStorage.

**Comment by lrembacz** [comment]
Yes, I did like that. Also I used not loading data on start property. 

The one more question I have. It would be nice if we could inject some additional props into details row. I had to do the sums on all details rows I opened and its imposible right now. I managed it with vuex and some hard logic on it but without it, we cant do the reactivity on all details rows at the time. It would be nice to have it there! Thanks for this package :D

**Comment by ratiw** [comment]
@lrembacz Can you show the image of your UI? I don't quite understand your use case, so I don't know how can make it relevant. Detail row has already got the row data, what kind of extra data would be relevant? The whole data set? Let's discuss this and maybe we can find something to add to.

**Comment by lrembacz** [comment]
![Screen](https://user-images.githubusercontent.com/32040964/36583904-ce9b3a50-1877-11e8-981d-53f2c128cd35.png)

Here I use this solution for counting sums depending on what the user has typed in the inputs. Thanks to this additional props, we could have data from individual details rows and work on them reactivly. Now I can verify the data or generate raport or anything I want. Its good solution for some advanced actions made for few rows at once.

**Comment by ratiw** [comment]
I can see that this is one use case that Vuetable is not built for. I mean if it were me, I wouldn't choose to use Vuetable for this case. It looks more like a "data form" than a data presentation UI. Please correct me if I'm wrong.

But for the sake of discussion (that might lead to something), how would you pass or map those additional data to the detail row as I can't see any other relationship other than the total of "sum amount"

**Comment by lrembacz** [comment]
Maybe you're right and this is the only use case that comes to my mind now. However, if we deal with data tables on economic data, what prevents us from implementing some more advanced operations instantly? I think that the data table in which you can perform collective operations on individual records from the database is a good solution. If you think otherwise, I do not have a problem with it. 

It was just a suggestion. 

A similar issue is related to joining several cells in a table, but of course you can also solve this directly via the api calls, but I think this actually use case is more like frontend that backend issue.
